src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__content">
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__img"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <div class="product-card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+      <div class="product-card__code">Product code: 195434</div>
+
+      <div class="product-card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star stars__star--active"></div>
+
+          <div class="stars__star stars__star--active"></div>
+
+          <div class="stars__star stars__star--active"></div>
+
+          <div class="stars__star stars__star--active"></div>
+
+          <div class="stars__star"></div>
```

Comment:
```suggestion
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```
Don't hardcode active stars, they should be changed using modifier of block 'stars'
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/c7170e87-192f-4bbe-93da-ed2205706759)